src/variables.py

```diff
@@ -78,3 +82,7 @@
 
 # List of ids of staking modules in which the depositor bot will make deposits
 DEPOSIT_MODULES_WHITELIST = [int(module_id) for module_id in os.getenv('DEPOSIT_MODULES_WHITELIST', '1').split(',')]
+
+
+def is_mellow_strategy_set() -> bool:
+    return SIMPLE_DVT_STAKING_STRATEGY_MELLOW_CONTRACT is not None
```

Comment:
This variable never be None, because os.getenv with not None default always return str.
And let's remove function from this file.

I propose to do
```python
MELLOW_CONTRACT_ADDRESS = os.getenv('MELLOW_CONTRACT_ADDRESS', None)
# Can be empty string - then skip
if MELLOW_CONTRACT_ADDRESS:
     # bot will throw exception if there some not expected str and it's ok
     MELLOW_CONTRACT_ADDRESS = Web3.to_checksum_address(MELLOW_CONTRACT_ADDRESS)
 
# -----
# in code
if MELLOW_CONTRACT_ADDRESS:
   ...
# or in lido_contracts.py
def is_mellow_enabled() -> bool:
     return bool(self.mellow_contract)
```

Reply:
Done with 
```
if MELLOW_CONTRACT_ADDRESS:
```